test/runtests.jl

```diff
@@ -262,6 +277,34 @@ cd(@__DIR__) do
                         test = popfirst!(tests)
                         running_tests[test] = now()
                         wrkr = p
+
+                        # Create a timer for this test to report long-running status
+                        test_timers[test] = Timer(longrunning_delay, interval=longrunning_interval) do timer
+                            if haskey(running_tests, test)  # Check test is still running
+                                start_time = running_tests[test]
+                                elapsed = now() - start_time
+                                elapsed_minutes = elapsed.value ÷ (1000 * 60)
+
+                                elapsed_str = if elapsed_minutes >= 60
+                                    hours = elapsed_minutes ÷ 60
+                                    mins = elapsed_minutes % 60
```

Comment:
Could simplify a tiny bit as
```suggestion
                                    hours, mins = divrem(elapsed_minutes, 60)
```